corehq/apps/userreports/management/commands/count_ucrs.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import textwrap
+from datetime import datetime
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from dimagi.utils.couch.database import iter_docs
+from corehq.dbaccessors.couchapps.all_docs import get_doc_ids_by_class
+from corehq.util.log import with_progress_bar
+
+from corehq.apps.userreports.models import ReportConfiguration
+
+
+class Command(BaseCommand):
+    help = "Pull stats about UCR and report-builder reports server-wide"
+
+    def handle(self, **options):
+        config_ids = get_doc_ids_by_class(ReportConfiguration)
+        total_count = len(config_ids)
+
+        builder_count, ucr_count = 0, 0
+        for doc in with_progress_bar(iter_docs(ReportConfiguration.get_db(), config_ids), total_count):
```

Comment:
i think this doesn't include static reports

Reply:
Yeah, that was semi-intentional - I'll check whether we want that (I think that's probably a 3rd category)

Reply:
also if you add it to admin reports, you could facet it on domain to see the domains with the most reports :trollface: